mantaray/commands.py

```diff
@@ -154,6 +156,30 @@ def kick(view: View, core: IrcCore, nick: str, reason: str | None = None) -> Non
         else:
             view.add_message("You can use /kick only on a channel.", tag="error")
 
+    def away(view: View, core: IrcCore, away_message: str | None = None) -> None:
+        if away_message is None:
+            current_view = view.server_view.irc_widget.get_current_view()
+            current_view.add_message(
+                "*",
+                (
+                    'Please use "/AWAY <away message>" to mark yourself as Away, or "/BACK" to remove your Away status.',
+                    ["error"],
+                ),
+            )
```

Comment:
You don't need this code, if you define `away_message: str` without a default value. Then `/away` with no message will produce an error.

Reply:
If I understand it correct, we still want to add this as a message to the user though so they know why their command is not working.

Therefore I'd say that the code I don't need is `if away_message is None`. I will instead use a `try/except`

Reply:
@Akuli Since your fixes `/away` is broken.
`/AWAY` gives message `Usage: /AWAY <away_message>`, but it doesn't have an error tag (which i think it should, to stand out in a busy channel).

`/AWAY Foo` doesn't clear the entry and you don't get the message that you are marked as being away. Same with `/BACK`.

Reply:
I understand now that you implemented a "Usage" error thingy. I would say this should have the error tag instead of info though. Not sure if all "Usage" messages should be errors though, so I'll leave this up to you.

Reply:
I created #239.